cloudferrylib/os/network/neutron.py

```diff
@@ -1081,81 +1102,70 @@ def add_router_interfaces(self, src_router, dst_router,
                 dst_router['id'],
                 {"subnet_id": ex_snet['id']})
 
-    def upload_floatingips(self, networks, src_floats):
-        LOG.info("Creating floating IPs")
-        existing_nets = self.get_networks()
-        ext_nets_ids = []
-        # getting list of external networks with allocated floating ips
-        for src_float in src_floats:
-            ext_net_id = self.get_new_extnet_id(src_float['floating_network_id'],
-                                                networks, existing_nets)
-            if ext_net_id not in ext_nets_ids:
-                ext_nets_ids.append(ext_net_id)
-                self.allocate_floatingips(ext_net_id)
-        existing_floatingips = self.get_floatingips()
-        self.recreate_floatingips(src_floats, networks,
-                                  existing_nets, existing_floatingips)
-        self.delete_redundant_floatingips(src_floats, existing_floatingips)
-
-    def allocate_floatingips(self, ext_net_id):
-        num_allocated_fips = 0
-        try:
-            while True:
-                LOG.debug("Creating floating IP %d on network '%s'",
-                          num_allocated_fips, ext_net_id)
-                self.neutron_client.create_floatingip({
-                    'floatingip': {'floating_network_id': ext_net_id}})
-                num_allocated_fips += 1
-        except neutron_exc.NeutronClientException as e:
-            if e.status_code == 409:  # 409 - Conflict
-                LOG.info("{0:d} floating IPs were allocated on network {1:s}"
-                         .format(num_allocated_fips, ext_net_id))
-            else:
-                raise
+    def _migration_tenant(self):
```

Comment:
Yeah, I've seen, thanks) It is called `_migration_tenant ` and returns `tenant_id` and it little bit confused me. So, for me it is not obvious what should do this method and where. Should it just define `tenant_id` of migrated tenant and that's all?